ui/learn/src/storage.ts

```diff
@@ -59,5 +70,10 @@ export default function (d?: LearnProgress): Storage {
         location.reload();
       }
     },
+    partial_reset: function (stageKey: string) {
+      xhrPartialReset(stageKey).then(function () {
+        location.reload();
+      });
```

Comment:
Stylistic choice but I'd use `() => location.reload()` here (and also adjust the usage above).